csaf_2.1/prose/edit/src/schema-elements-02-props-03-vulnerabilities.md

```diff
@@ -370,6 +370,14 @@ Vulnerability notes (`notes`) of value type Notes Type (`notes_t`) holds notes a
     },
 ```
 
+The following combinations of `category` and `title` have a special meaning and MUST be used as stated below:
+
+| `category` | `title` | content of `text` |
+|---------------|---------------|-------------------|
+| `description` | CVE description | Contains the official and unchanged CVE description for this specific vulnerability. |
```

Comment:
"Title Case" for title cells (CVE Description)?

Reply:
done